P0009/P0009.md

```diff
@@ -2322,7 +2319,7 @@ public:
   using reference = typename accessor_type::reference;
 
   // [mdspan.basic.cons], mdspan constructors, assignment, and destructor
-  constexpr mdspan() = default;
+  constexpr mdspan() requires(rank_dynamic != 0) = default;
```

Comment:
This differs from `span` in that it permits default construction if any extent is (statically) zero.  Is that intentional?  I'm guessing so (I would want an `mdspan` with any static extent of zero to be default constructible), but just checking.

Reply:
ah hm ok I can change that.